README.md

```diff
@@ -1,82 +1,83 @@
 # @knapsack-pro/jest
 
-[![CircleCI](https://circleci.com/gh/KnapsackPro/knapsack-pro-jest.svg?style=svg)](https://circleci.com/gh/KnapsackPro/knapsack-pro-jest)
+<p align="center">
+  <a href="https://knapsackpro.com?utm_source=github&utm_medium=readme&utm_campaign=knapsack-pro-jest&utm_content=hero_logo">
+    <img alt="Knapsack Pro" src="./.github/assets/knapsack.png" width="300" height="300" style="max-width: 100%;" />
+  </a>
+</p>
 
-`@knapsack-pro/jest` runs your tests with [Jest](https://jestjs.io) test runner and does dynamic tests allocation across parallel CI nodes using [KnapsackPro.com](https://knapsackpro.com?utm_source=github&utm_medium=readme&utm_campaign=%40knapsack-pro%2Fjest&utm_content=sign_up) Queue Mode to provide the fastest CI build time (optimal test suite timing).
+<h3 align="center">Speed up your tests</h3>
+<p align="center">Run your 1-hour test suite in 2 minutes with optimal parallelisation on your existing CI infrastructure</p>
 
-Learn about Knapsack Pro Queue Mode in the video [how to run tests with dynamic test suite split](https://youtu.be/hUEB1XDKEFY) and learn what CI problems can be solved thanks to it.
+---
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+<div align="center">
+  <a href="https://circleci.com/gh/KnapsackPro/knapsack-pro-jest">
+    <img alt="Circle CI" src="https://circleci.com/gh/KnapsackPro/knapsack-pro-jest.svg?style=svg" />
+  </a>
+</div>
 
-## Table of Contents
+<br />
+<br />
 
-- [Installation](#installation)
-- [FAQ](#faq)
-- [Development](#development)
-  - [Requirements](#requirements)
-  - [Dependencies](#dependencies)
-  - [Setup](#setup)
-  - [Publishing](#publishing)
-  - [Testing](#testing)
-    - [CI](#ci)
-    - [Example Jest test suite](#example-jest-test-suite)
+Knapsack Pro wraps [Jest](https://jestjs.io) and works with your existing CI infrastructure to parallelize tests optimally:
 
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+- Dynamically splits your tests based on up-to-date test execution data
+- Is designed from the ground up for CI and supports all of them
+- Tracks your CI builds to detect bottlenecks
+- Does not have access to your source code and collects minimal test data (with opt-in encryption)
```

Comment:
> (with opt-in encryption)

This feature is not available for Jest and Cypress yet
Would be nice to add it at some point.

# Related issue
https://github.com/KnapsackPro/knapsack-pro-jest/issues/32

Reply:
Removed. Thanks for checking 🙏